jetstream/resource_jetstream_consumer.go

```diff
@@ -327,6 +334,16 @@ func consumerConfigFromResourceData(d *schema.ResourceData) (cfg api.ConsumerCon
 		cfg.AckPolicy = api.AckNone
 	}
 
+	fs, ok := d.GetOk("filter_subjects")
+	if ok {
+		ar := fs.([]any)
+		var subjects = make([]string, len(ar))
+		for i, v := range ar {
+			subjects[i] = v.(string)
+		}
+		cfg.FilterSubjects = subjects
```

Comment:
we have to do a bit more here I am afraid, see this:

https://github.com/nats-io/natscli/blob/875003bdaf4b55a263b1221cf1a64d1b9d482412/cli/consumer_command.go#L376-L382

Reply:
@ripienaar PR updated. `filter_subject` and `filter_subjects` are now exclusive.